src/components/quickPostModal/quickPostModalContent.tsx

```
@@ -126,11 +126,15 @@ export const QuickPostModalContent = forwardRef(
-      handleSheetClose() {
-        _addQuickCommentIntoCache();
-      },
-    }));
+    useImperativeHandle(
+      ref,
+      () => ({
+        handleSheetClose() {
+          _addQuickCommentIntoCache(commentValueRef.current);
+        },
+      }),
+      [_addQuickCommentIntoCache],
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)